patcher update

lubiarz-oliwia opened the pull request:
patcher page, early access form page, great news page, how it works page update, landing zone page update, reference architecture page update, archive compliance page update, pricing page update

lubiarz-oliwia committed:
patcher page, early access form page, great news page, how it works page update, landing zone page update, reference architecture page update, archive compliance page update, pricing page update

ArkuVonSymfon commented on _data/initial-setup-how-it-works.yml:
```diff
@@ -37,3 +37,11 @@
+    <p>
+      Early access to Patcher is just a few clicks away with CIS AWS Foundations Benchmark. 
+      Our CLI tool automates the entire process of updating code dependencies, allowing you to stay up-to-date without any hassle. 
+      <a href="/patcher" target="_blank">Check out how simple it is.</a>
```
Just for double check if `/patcher` should be opened in a new window / tab

ArkuVonSymfon commented on _data/patcher-features.yml:
```diff
@@ -0,0 +1,11 @@
+- title: Commercially-maintained infrastructure code
+  description: We keep the Infrastructure as Code Library and Reference Architecture up to date with the latest best practices, security releases, tool versions (e.g., latest versions of Terraform, Kubernetes, EKS), and compliance standards (e.g., latest versions of the AWS CIS Foundations Benchmark).
```
NIT: Break line

ArkuVonSymfon commented on pages/patcher/_hero.html:
```diff
@@ -0,0 +1,32 @@
+              #ff4f47 98.56%
+            );
+            margin-left: 12px;
+          " class="label-beta label-small">
```
break line after "

ArkuVonSymfon commented on pages/patcher/_hero.html:
```diff
@@ -0,0 +1,32 @@
+      </p>
+    </div>
+    <div class="col-lg-7">
+      <img class="floating-img-patcher" src="/assets/img/patcher/hero@3x.png"
```
```suggestion
      <img 
          class="floating-img-patcher"
          src="/assets/img/patcher/hero@3x.png"
```

ArkuVonSymfon commented on pages/patcher/_how-it-works.html:
```diff
@@ -0,0 +1,80 @@
+</div>
+{% endfor %} {% endfor %}
+
+<script>
```
every JS script should be started after the document so add "document ready" or "on load"

ArkuVonSymfon commented on pages/pricing/_standard-pricing.html:
```diff
@@ -2,45 +2,104 @@
+      </p>
     </div>
-    <hr>
+    <hr />
```
in HTML5 we don't need `/` in `<hr>` or `<br>` please check other examples in project but I think it can be as `<hr>`

lubiarz-oliwia committed:
add review suggested changes

klijakub committed:
Merge branch 'gruntwork-io:master' into patcher-update

josh-padnick commented:
This looks pretty good! I have just a few small final requests and then I think we're set!

1. In the webform, the checkboxes can only be checked if you click on the checkbox, but not the text associated with the checkbox. Could you update it so that when you click on the text, it selects/deselects the associated checkbox?

    <img width="501" alt="image" src="https://user-images.githubusercontent.com/4295964/236967726-4c80ad0d-d5b7-402c-b626-d517499805d9.png">

2. Could you add a field for "Title" after "Name" and before "Company Email Address"
3. "Company email adress" is misspelled. It should be "Company Email Address" (with two d's)

lubiarz-oliwia committed:
add review suggested changes

lubiarz-oliwia committed:
add review suggested changes

lubiarz-oliwia committed:
add review suggested changes

josh-padnick committed:
Various clean up

- Add company field
- Standardize naming conventions
- Update answers to match available options in HubSpot

josh-padnick commented:
Folks, I think we're all set here! I'm pushing this live now!
